dask_kubernetes/operator/tests/test_operator.py

```diff
@@ -31,6 +33,13 @@ async def simplecluster(k8s_cluster, operator):
 
     yield cluster_name
 
+    # Check that the schedeuler pod and service exist
+    scheduler_pod_name = "simple-cluster-scheduler"
+    scheduler_service_name = "simple-cluster"
+
+    assert scheduler_pod_name in k8s_cluster.kubectl("get", "pods")
+    assert scheduler_service_name in k8s_cluster.kubectl("get", "services")
+
```

Comment:
This should be in `test_simplecluster`, not the fixture. 

Also it is failing because there isn't enough time for the pod to be created before we do `get pods` here. You could turn each of these into a `while`.

```python
while scheduler_pod_name not in k8s_cluster.kubectl("get", "pods"):
    await asyncio.sleep(0.1)
```

If the pod never gets created the test will timeout at 60 seconds and fail.